example_js_standalone_smart_card_client_app/background.js

```diff
@@ -87,6 +87,29 @@ function listReaders() {
 /** @param {!Array.<string>} readers List of reader names. */
 function onReadersListed(readers) {
   console.log('List of PC/SC-Lite readers: ' + readers);
+  if (readers.length)
+    waitForReaderCardInserted(readers[0]);
+}
+
+/** @param {string} reader Name of the reader */
+function waitForReaderCardInserted(reader) {
+  api.SCardGetStatusChange(
+         sCardContext, GoogleSmartCard.PcscLiteClient.API.INFINITE, [{
+           reader_name: reader,
+           current_state: GoogleSmartCard.PcscLiteClient.API.SCARD_STATE_EMPTY
```

Comment:
Can we just assume that there is no card (recognized) at this point?
The way I read the API, calling SCardGetStatusChange requires knowledge of the current reader state, or otherwise using SCARD_STATE_UNAWARE instead.

Reply:
Done - You're right, it's more standard that way.